app/controllers/RegularContributions.scala

```diff
@@ -115,8 +113,8 @@ class RegularContributions(
     SafeLogger.info(s"[${request.uuid}] User ${fullUser.id} is attempting to create a new ${request.body.contribution.billingPeriod} contribution")
 
     val result = for {
-      user <- identityService.getUser(fullUser) // FIXME also need to check they are email verified before using AccessScopeNoRestriction
-      response <- client.createContributor(request, contributor(user, request.body), request.uuid, AuthenticatedAccess).leftMap(_.toString)
+      user <- identityService.getUser(fullUser)
```

Comment:
if they are logged in, we will have already asked them on the client side to go and amend their existing contribution.  So by this time we just go ahead anyway.